fe/fe-core/src/main/java/com/starrocks/system/ComputeNode.java

```diff
@@ -116,6 +118,10 @@ public class ComputeNode implements IComputable, Writable {
     @SerializedName("isSetStoragePath")
     private volatile boolean isSetStoragePath = false;
 
+    // Tracking the heartbeat status, CONNECTING/ALIVE/SHUTDOWN/DISCONNECTED
+    @SerializedName("status")
+    private Status status;
```

Comment:
Is the string literal of enum item serialized? That means the name can't be changed anymore. Better to add NOTE to the enum definition.